pymc/tests/distributions/test_continuous.py

```diff
@@ -0,0 +1,2229 @@
+                np.linspace(0, 10, 10),
+                st.norm.pdf(np.linspace(0, 10, 10), loc=2.5, scale=1),
+                None,
+                2.5270134,
+            ),
+            (
+                np.linspace(0, 10, 100),
+                st.norm.pdf(np.linspace(0, 10, 100), loc=2.5, scale=1),
+                None,
+                2.51771721,
+            ),
+        ],
+    )
+    def test_interpolated_moment(self, x_points, pdf_points, size, expected):
+        with pm.Model() as model:
+            pm.Interpolated("x", x_points=x_points, pdf_points=pdf_points, size=size)
+        assert_moment_is_expected(model, expected)
+
+    @pytest.mark.parametrize(
+        "mu, sigma, size, expected",
+        [
+            (4.0, 3.0, None, 7.8110885363844345),
+            (4.0, np.full(5, 3), None, np.full(5, 7.8110885363844345)),
+            (np.arange(5), 1, None, np.arange(5) + 1.2703628454614782),
+            (np.arange(5), np.ones(5), (2, 5), np.full((2, 5), np.arange(5) + 1.2703628454614782)),
+        ],
+    )
+    def test_moyal_moment(self, mu, sigma, size, expected):
+        with pm.Model() as model:
+            pm.Moyal("x", mu=mu, sigma=sigma, size=size)
+        assert_moment_is_expected(model, expected)
+
+    @pytest.mark.parametrize(
+        "alpha, mu, sigma, size, expected",
+        [
+            (1.0, 1.0, 1.0, None, 1.56418958),
+            (1.0, np.ones(5), 1.0, None, np.full(5, 1.56418958)),
+            (np.ones(5), 1, np.ones(5), None, np.full(5, 1.56418958)),
+            (
+                np.arange(5),
+                np.arange(1, 6),
+                np.arange(1, 6),
+                None,
+                (1.0, 3.12837917, 5.14094894, 7.02775903, 8.87030861),
+            ),
+            (
+                np.arange(5),
+                np.arange(1, 6),
+                np.arange(1, 6),
+                (2, 5),
+                np.full((2, 5), (1.0, 3.12837917, 5.14094894, 7.02775903, 8.87030861)),
+            ),
+        ],
+    )
+    def test_skewnormal_moment(self, alpha, mu, sigma, size, expected):
+        with pm.Model() as model:
+            pm.SkewNormal("x", alpha=alpha, mu=mu, sigma=sigma, size=size)
+        assert_moment_is_expected(model, expected)
+
+
+@pytest.mark.parametrize(
+    "b, kappa, mu, size, expected",
+    [
+        (1.0, 1.0, 1.0, None, 1.0),
+        (1.0, np.ones(5), 1.0, None, np.full(5, 1.0)),
+        (np.arange(1, 6), 1.0, np.ones(5), None, np.full(5, 1.0)),
+        (
+            np.arange(1, 6),
+            np.arange(1, 6),
+            np.arange(1, 6),
+            None,
+            (1.0, 1.25, 2.111111111111111, 3.0625, 4.04),
+        ),
+        (
+            np.arange(1, 6),
+            np.arange(1, 6),
+            np.arange(1, 6),
+            (2, 5),
+            np.full((2, 5), (1.0, 1.25, 2.111111111111111, 3.0625, 4.04)),
+        ),
+    ],
+)
+def test_asymmetriclaplace_moment(b, kappa, mu, size, expected):
```

Comment:
This one is missing an indentation, which is causing the following tests to be inside it and not run (codecov warning)